Fix: Add overflow hidden class to body when bottom-sheet opened

cagataycali opened the pull request:
This PR fixes the body's scrolling when bottom-sheet component opened.
Listens componentWillUpdate; if the case of opening or closing component, toggle the class of fixer of overflow.

yavuzkoca commented on src/components/bottom-sheet/bottom-sheet.tsx:
```diff
@@ -12,6 +12,26 @@ export default class BottomSheet extends PureComponent<IBottomSheetProps> {
+      document.body.classList.add("q-disable-scroll");
+    }
+  }
+
```
Please also remove this class in componentWillUnMount life cycle

cagataycali replied:
Resolved

cagataycali commented:
Also added missing css properties into modal.scss for further bugs;

> If we gonna use modal w/o existing popup component in body q-disable-scroll css class does not exists in css bundle. PR also solve this bug. 

```css
body.q-disable-scroll {
  position: fixed;
  overflow: hidden;
  height: 100%;
  width: 100%;
}

html.q-disable-scroll {
  overflow: hidden;
  height: 100%;
  width: 100%;
}
```
